universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -33,6 +33,19 @@
 import services.question.types.QuestionDefinitionBuilder;
 import services.question.types.QuestionType;
 
+/**
+ * An EBean mapped class that stores the configuration for a single question.
+ *
+ * <p>Questions can be shared by multiple {@code Program}s. The set of questions for a given program
```

Comment:
nit: "for a given program" sounds a little misleading to me. But the i.e. makes sense! Maybe "The set of questions for all programs at a given version"?

Reply:
oh good catch that was meant to say "version" instead of "program"